processor/trace.txt

```diff
@@ -0,0 +1 @@
+{"exec":[{"clk":0,"pc":0,"fp":0,"instruction":{"JMP":{"a":{"Immediate":6}}},"regs":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":1,"pc":6,"fp":1,"instruction":{"MOV":{"ri":5,"a":{"Immediate":3}}},"regs":[0,0,0,0,0,3,0,0,0,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":2,"pc":7,"fp":1,"instruction":{"ADD":{"ri":5,"rj":5,"a":{"Immediate":5}}},"regs":[0,0,0,0,0,8,0,0,0,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":3,"pc":8,"fp":1,"instruction":{"MOV":{"ri":1,"a":{"RegName":5}}},"regs":[0,8,0,0,0,8,0,0,0,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":4,"pc":9,"fp":1,"instruction":{"MOV":{"ri":2,"a":{"Immediate":7}}},"regs":[0,8,7,0,0,8,0,0,0,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":5,"pc":10,"fp":1,"instruction":{"JMP":{"a":{"Immediate":1}}},"regs":[0,8,7,0,0,8,0,0,0,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":6,"pc":1,"fp":11,"instruction":{"ADD":{"ri":0,"rj":1,"a":{"RegName":2}}},"regs":[15,8,7,0,0,8,0,0,0,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":7,"pc":2,"fp":11,"instruction":{"MUL":{"ri":3,"rj":0,"a":{"Immediate":2}}},"regs":[15,8,7,30,0,8,0,0,0,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":8,"pc":3,"fp":11,"instruction":{"MOV":{"ri":4,"a":{"RegName":3}}},"regs":[15,8,7,30,30,8,0,0,0,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":9,"pc":4,"fp":11,"instruction":{"MOV":{"ri":8,"a":{"RegName":4}}},"regs":[15,8,7,30,30,8,0,0,30,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":10,"pc":5,"fp":11,"instruction":{"RET":{"ri":8}},"regs":[15,8,7,30,30,8,0,0,30,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":11,"pc":11,"fp":1,"instruction":{"MOV":{"ri":6,"a":{"RegName":8}}},"regs":[15,8,7,30,30,8,30,0,30,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":12,"pc":12,"fp":1,"instruction":{"MUL":{"ri":7,"rj":4,"a":{"RegName":4}}},"regs":[15,8,7,30,30,8,30,900,30,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":13,"pc":13,"fp":1,"instruction":{"MOV":{"ri":4,"a":{"RegName":7}}},"regs":[15,8,7,30,900,8,30,900,30,0,0,0,0,0,0,0],"flag":false,"v_addr":null},{"clk":14,"pc":14,"fp":1,"instruction":{"END":[]},"regs":[15,8,7,30,900,8,30,900,30,0,0,0,0,0,0,0],"flag":false,"v_addr":null}]}
```

Comment:
json

Reply:
ok. I will delete this file. It is a intermediate file.